File: Code-Chef/MEDEL.cpp
/*https://www.codechef.com/COOK111B/problems/MDL/*/
/*AUthor:-invictus1599*/
#include <iostream>
using namespace std;
int del(int a[],int x,int n){
	int i;
	if(x<=n-2){
		for(i=x;i<n;i++)
			a[i]=a[i+1];
	}
}
int main() {
	int t;
	cin>>t;
	while(--t>=0){
	    int n,i,t;
	    cin>>n;
	    int a[n];
	    t=n;
	    for(i=0;i<n;i++)cin>>a[i];
	    while(t>=3){
	    	if(a[0]>=a[1] && a[0]>=a[2]){
	    		if(a[1]>=a[2])
	    			del(a,1,t);
	    		else
	    			del(a,2,t);
			}
			else if(a[1]>=a[2]){
				if(a[0]>=a[2])
					del(a,0,t);
				else
					del(a,2,t);
			}
			else{
				if(a[1]>=a[0])
					del(a,1,t);
				else
					del(a,0,t);
			}
			t--;
		}
	    for(i=0;i<2;i++)cout<<a[i]<<" ";
		cout<<endl;
	}
	return 0;
}

File: Code-Chef/Medel/medel.py
'''
Problem Statement: https://www.codechef.com/problems/MDL
Author: striker
'''

def main():
    test = int(input())
    for _ in range(test):
        n = int(input())
        sequence = list(map(int,input().split()))
        max_val = max(sequence) ; min_val = min(sequence)
        max_val_index = sequence.index(max_val) ; min_val_index = sequence.index(min_val)
        if max_val_index < min_val_index:
            print(str(max_val) + " " + str(min_val))
        else:
            print(str(min_val) + " " + str(max_val))

if __name__ == "__main__":
    main()